beacon-import.yml

```diff
@@ -4,10 +4,22 @@
   hosts:
     - beacon_import
   vars_files:
+    - "secret_group_vars/all.yml"
     - secret_group_vars/beacon.yml
   vars:
     script_user: beacon
     script_dir: /home/beacon/script
     galaxy_api_url: https://usegalaxy.eu
   roles:
+    - role: usegalaxy_eu.handy.os_setup
+      vars:
+        enable_hostname: true
+        enable_powertools: true        # geerlingguy.repo-epel role doesn't enable PowerTools repository
+    - geerlingguy.repo-epel     # Install EPEL repository
+    - usegalaxy-eu.autoupdates  # keep all of our packages up to date
+    - influxdata.chrony
+    - dj-wasabi.telegraf
+    - usegalaxy-eu.dynmotd # nicer MOTD/welcome message
     - paprikant.beacon-importer
+    - os_hardening
```

Comment:
for roles `os_hardening` and `ssh_hardening` I think you need to include the `collections`

```
  collections:
    - devsec.hardening
```